<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgColor</class>
 <widget class="QDialog" name="DlgColor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="LabelTitle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>91</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>LabelTitle</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelSetting">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>0</y>
     <width>231</width>
     <height>91</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>LabelSetting</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelStatusBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>420</y>
     <width>641</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>LabelStatusBar</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelPageNum">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>430</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>LabelPageNum</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelSet2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>LabelSet2</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelText14">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>170</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>LabelText14</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelText2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>LabelText2</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelText13">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>LabelText13</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelSet4">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>250</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>LabelSet4</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelText11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>LabelText11</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelSet3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>250</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>LabelSet3</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelSet1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>LabelSet1</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelText4">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>250</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>LabelText4</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelText3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>250</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>LabelText3</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelText12">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>LabelText12</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelText1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>LabelText13</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
